astropy/utils/iers/iers.py

```diff
@@ -443,5 +473,49 @@ def pm_source(self, i):
         """Set PM source flag for entries in IERS table"""
         return np.ones_like(i) * FROM_IERS_B
 
+class IERS_Auto(IERS_A):
```

Comment:
I don't understand why a separate class instead of a new argument to the existing class's `open` method.